ui/components/Heading.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import styled from "styled-components";
+
+type Props = {
+  className?: string;
+  level: 1 | 2 | 3 | 4 | 5;
+  children: any;
+};
+
+function Heading({ className, children, level }: Props) {
+  return (
+    <div className={className}>
+      {React.createElement(`h${level}`, { className }, children)}
+    </div>
```

Comment:
I know I'm being so annoying - does it have to be in a div? Otherwise I like the idea that we have the standard headers that will automatically be the right size and style so we don't have to think about it

Reply:
Fixed